charts/refractr/templates/issuer.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: letsencrypt
```

Comment:
As I understand, this issuer is only for refractr. In that case, shouldn't it be in the refractr namespace, or in the cert-manager namespace? Setting explicitly a namespace would make the deployment more reliable, since that part won't go to default or to the current active ns.

Reply:
So this is a part I don't fully understand yet but I've not put the namespace anywhere because my impression was that you specify it when installing the helm chart.  And hence the helm chart is generic enough to reuse anywhere.  When installing via the helm cli I specify the namespace there as well as which Let's Encrypt url to use (prod or stage).  When we switch to the Helm Operator, I envisioned all that being set in yaml.  Feel free to enlighten me!  But yes this is an issuer specific to the refractr namespace, it doesn't go in the cert-manager namespace.  And though cert-manager has cluster issuers, I was shying away from them for better isolation.

Reply:
ok, this makes sense. Yes somehow Helm already applies all manifest to the same namespace. 
The approach I took was to define a variable for a namespace and reference it in all the manifests.
Your approach is cleaner and I like it more. I'll be removing that from my manifests. Thanks!